app/views/errors/not_found.html.erb

```diff
@@ -0,0 +1,135 @@
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>The page you requested was not found</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Cabin:400,700"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Montserrat:900"
+      rel="stylesheet"
+    />
+    <link type="text/css" rel="stylesheet" href="css/style.css" />
+    <style>
```

Comment:
i am not familiar with error pages, but you have errors_pages.scss file above, better to move styles there if it possible.